builtin/providers/aws/resource_aws_cloudformation_stack.go

```diff
@@ -385,6 +393,10 @@ func resourceAwsCloudFormationStackUpdate(d *schema.ResourceData, meta interface
 		input.StackPolicyURL = aws.String(d.Get("policy_url").(string))
 	}
 
+	if d.HasChange("iam_role_arn") {
+		input.RoleARN = aws.String(d.Get("iam_role_arn").(string))
```

Comment:
I don't believe so - it's the same bug as always AFAICT - when we stop managing something with Terraform, Terraform doesn't recognise that it has "changed"